notebooks/Analyze_Model_Outputs.ipynb

```diff
@@ -0,0 +1,3047 @@
+{
```

Comment:
<p>nit: dataframes, Dataframe -&gt; DataFrame</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Fixed.</p>